### PR TITLE
Fix bug where tags where not correctly displayed

### PR DESCRIPTION
**Pull request by Alberto-o**
Adds a type taglist_t to hold directional information of tags, needed to visualized them in vtk

### CODE REVIEW

File: src_main_pub/tag_changes.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+launch_simulation
```
Comment (lmdiazangulo) [inline src_main_pub/tag_changes.txt:1]: Is this file added by mistake?

File: test/pyWrapper/utils.py [hunk 2]
```diff
@@ -3,8 +3,13 @@
 import json
 import numpy as np
 import matplotlib.pyplot as plt
+<<<<<<< HEAD
+import pyvista as pv
```
Comment (lmdiazangulo) [inline test/pyWrapper/utils.py:7]: There is an error here. An incomplete merge

**Review by lmdiazangulo — CHANGES_REQUESTED** [review]

File: src_main_pub/tag_changes.txt (hunk 1 above)
Comment (Alberto-o) [reply src_main_pub/tag_changes.txt]: Yes, removed

File: test/pyWrapper/utils.py (hunk 2 above)
Comment (Alberto-o) [reply test/pyWrapper/utils.py:6]: Fixed

**Review by lmdiazangulo — APPROVED** [review]

**Pull request merged**